src/convenience/frameworks.node.ts

```diff
@@ -1,90 +1,85 @@
 // deno-lint-ignore-file no-explicit-any
-import type { IncomingMessage, ServerResponse } from "http";
+import { IncomingMessage, ServerResponse } from "http";
```

Comment:
Prefer `import type`. This will allows to not import the module here at runtime.